pkgs/development/libraries/libossp-uuid/default.nix

```diff
@@ -12,6 +12,12 @@ stdenv.mkDerivation {
 
   configureFlags = stdenv.lib.optional stdenv.isFreeBSD "--with-pic";
 
+  patches = [ ./shtool.patch ];
+
+  preConfigure = ''
+  export ac_cv_va_copy=yes
```

Comment:
I think conventionally in Nixpkgs, this sort of stuff is not exported but added to `configureFlags` instead.

Reply:
Thats the problem. This `configure` script does not allow you to fix the test results on `va_copy`. Please advise a better approach, if any.